README.md

```diff
@@ -63,13 +68,38 @@ apolloFetch
   .use(middleware3);
 ```
 
+### Custom Fetch
+
 For mocking and other fetching behavior, you may pass a fetch into `createApolloFetch`:
 
 ```js
 const customFetch = createFileFetch();
 const apolloFetch = createApolloFetch({ customFetch });
 ```
 
+### Custom GraphQL to Fetch Translation
+
+To modify how GraphQL requests are incorporated in the fetch options, you may pass a transformation function into `createApolloFetch`.
+`apollo-fetch` exports `constructDefaultOptions` to allow conditional creation of the fetch options.
+These transformations can be useful for servers that have different formatting for batches or other extra capabilities.
+
+```js
+const constructOptions = (requestOrRequests, init) => {
+  return {
+    ...init,
+    body: JSON.stringify(requestOrRequests),
+  }
+};
+const apolloFetch = createApolloFetch({ constructOptions });
+
+//simplified usage inside apolloFetch
+fetch(uri, constructOptions(request, options)) //options are result of middleware
```

Comment:
For this section, I think it could be helpful to include (not inline) the type definitions of `request` and `options` so developers can know what will be present when creating their own functions